scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/client/SchedulerNodeClient.java

```diff
@@ -0,0 +1,436 @@
+        System.out.println("[SchedulerNodeClient] Connected");
+    }
+
+    public List<JobUsage> getMyAccountUsage(Date startDate, Date endDate) throws NotConnectedException, PermissionException {
+        return client.getMyAccountUsage(startDate, endDate);
+    }
+
+    public List<JobUsage> getAccountUsage(String user, Date startDate, Date endDate) throws NotConnectedException, PermissionException {
+        return client.getAccountUsage(user, startDate, endDate);
+    }
+
+    public List<String> getUserSpaceURIs() throws NotConnectedException, PermissionException {
+        return client.getUserSpaceURIs();
+    }
+
+    public List<String> getGlobalSpaceURIs() throws NotConnectedException, PermissionException {
+        return client.getGlobalSpaceURIs();
+    }
+
+    public JobResult getJobResult(JobId jobId) throws NotConnectedException, PermissionException, UnknownJobException {
+        return client.getJobResult(jobId);
+    }
+
+    public TaskResult getTaskResultFromIncarnation(JobId jobId, String taskName, int inc) throws NotConnectedException, UnknownJobException, UnknownTaskException, PermissionException {
+        return client.getTaskResultFromIncarnation(jobId, taskName, inc);
+    }
+
+    public boolean killTask(JobId jobId, String taskName) throws NotConnectedException, UnknownJobException, UnknownTaskException, PermissionException {
+        return client.killTask(jobId, taskName);
+    }
+
+    public boolean restartTask(JobId jobId, String taskName, int restartDelay) throws NotConnectedException, UnknownJobException, UnknownTaskException, PermissionException {
+        return client.restartTask(jobId, taskName, restartDelay);
+    }
+
+    public boolean preemptTask(JobId jobId, String taskName, int restartDelay) throws NotConnectedException, UnknownJobException, UnknownTaskException, PermissionException {
+        return client.preemptTask(jobId, taskName, restartDelay);
+    }
+
+    public boolean removeJob(JobId jobId) throws NotConnectedException, UnknownJobException, PermissionException {
+        return client.removeJob(jobId);
+    }
+
+    public void listenJobLogs(JobId jobId, AppenderProvider appenderProvider) throws NotConnectedException, UnknownJobException, PermissionException {
+        client.listenJobLogs(jobId, appenderProvider);
+    }
+
+    public boolean killJob(JobId jobId) throws NotConnectedException, UnknownJobException, PermissionException {
+        return client.killJob(jobId);
+    }
+
+    public boolean pauseJob(JobId jobId) throws NotConnectedException, UnknownJobException, PermissionException {
+        return client.pauseJob(jobId);
+    }
+
+    public boolean resumeJob(JobId jobId) throws NotConnectedException, UnknownJobException, PermissionException {
+        return client.resumeJob(jobId);
+    }
+
+    public void changeJobPriority(JobId jobId, JobPriority priority) throws NotConnectedException, UnknownJobException, PermissionException, JobAlreadyFinishedException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1161)